### PR TITLE
Adds support for the "return" method

### PR DESCRIPTION
**Pull request by nicholasjhenry**
https://www.webpurify.com/documentation/methods/return/

Currently, the base is of #4  as I needed this to have `mix test.watch` running.

### CODE REVIEW

File: lib/web_purifex/parser.ex [hunk 1]
```diff
@@ -31,7 +31,11 @@ defmodule WebPurifex.Parser do
       |> Kernel.||("0")
       |> String.to_integer
 
-    {:ok, %Response{status: status, found: found}}
+    expletive =
+      response_body
+      |> get_in(["rsp", "expletive"])
```
Comment (nicocharlery) [inline lib/web_purifex/parser.ex:36]: Would always returning a list (empty or not) make the usage of the library easier instead of `nil | binary | list` ?

Comment (nicholasjhenry) [reply lib/web_purifex/parser.ex:36]: :-) I had considered that but decided to stay as close to the API. I'm on the fence about changing the original API. Thoughts?

Comment (nicocharlery) [reply lib/web_purifex/parser.ex:36]: This is just an opinion (/2cents):
- an API should be as predictable as possible, so returning multiple types is a mistake to me
- we are doing an abstraction anyway by building this library, we can leverage that to fix the smells

:)

Comment (nicholasjhenry) [reply lib/web_purifex/parser.ex:36]: Well, since we both considered this, probably suggests returning a single type is a good way to go. Will update. 👍

**Comment by nicholasjhenry** [comment]
@bokay ready for round two :-)

**Review by nicocharlery — APPROVED** [review]
🚀 Good to go!

**Pull request merged**